css/ampshared.css

```diff
@@ -155,6 +155,16 @@ i-amphtml-sizer {
   display: block !important;
 }
 
+/*
+ * Disable `i-amphtml-sizer` when aspect ratio is supported by both the
+ * borwser and the optimizer.
+ */
+@supports (aspect-ratio: 1 / 1) {
+  i-amphtml-sizer.i-amphtml-disable-for-ar {
+    display: none !important;
```

Comment:
I think this is actually a regression. We took SSR rendering with correct size, and his the thing that gave it size. We now have to wait for v0 to init, define, and connect the CE for size to come back.

Reply:
The idea is that the optimizer will do two things:
1. Set `style="aspect-ratio: width / height"`.
2. Add `i-amphtml-disable-for-ar` class to the sizer.

Runtime will only disable `i-amphtml-disable-for-ar` when it supports `aspect-ratio`. Thus there shouldn't be any regressions. Optimizer will simply gives us two "correct" sizes and we don't need them both.

Reply:
Ahh, ok.